code/game/objects/items/devices/tablets.dm

```diff
@@ -165,7 +162,11 @@
 	if(action == "switch_camera")
 		var/c_tag = params["name"]
 		var/list/cameras = get_available_cameras()
-		var/obj/machinery/camera/selected_camera = cameras[c_tag]
+		var/obj/machinery/camera/selected_camera
+		for(var/obj/machinery/camera/camera as anything in cameras)
+			if(camera.c_tag == c_tag)
+				selected_camera = camera
+				break
```

Comment:
why did you change this to not be an assoc lookup?

Reply:
@TiviPlus Bugs out with duplicate names and was missing out on multiple mobs for some reason

Reply:
good idea actually